apps/artemis-web/pages/hijack.tsx

```diff
@@ -85,7 +85,8 @@ const ViewHijackPage = (props) => {
 
   useGraphQl('hijackByKey', {
     callback: (data) => {
-      const hijacks = data.subscriptionData.data.view_hijacks;
+
+      const hijacks = data.subscriptionData ? data.subscriptionData.data.view_hijacks : data.view_hijacks;
```

Comment:
what does this offer? If we do not have `subscriptionData` then what do `data.view_hijacks` contain actually?

Reply:
the same thing :) its websockets vs rest :)